src/Core.ts

```diff
@@ -65,7 +65,7 @@ interface PackageJSON {
   readonly homepage: string
 }
 
-const PackageJSONDecoder = pipe(TD.type({ name: TD.string }), TD.intersect(TD.partial({ homepage: TD.string })))
+const PackageJSONDecoder = pipe(TD.struct({ name: TD.string }), TD.intersect(TD.partial({ homepage: TD.string })))
```

Comment:
Due to deprecation

Reply:
This will break though because the `struct` combinator was added in `io-ts@2.2.15` and `docs-ts` is currently using `io-ts@2.2.13`. Moreover, this is currently failing in CI because the `struct` combinator does not exist.

@gcanti - I am not sure if bumping `io-ts` would be considered a breaking change?

Reply:
Yea - I would leave it as `type` for now so we can avoid a (potentially) breaking change 👍 .

Reply:
So I did and CI passes now

Reply:
> I am not sure if bumping io-ts would be considered a breaking change?

@IMax153 `io-ts` is listed in `dependencies` so no, I don't think it would be a breaking change if we upgrade its version